draft-ietf-httpbis-digest-headers.md

```diff
@@ -1102,6 +1102,18 @@ or by sending multiple representation-data-digest values from which the receiver
 Endpoints are advised that sending multiple values consumes resources,
 which may be wasted if the receiver ignores them (see {{digest}}).
 
+## Resource exhaustion
+
+`Digest` validation consumes computational resources.
+To mitigate the risk of resource exhaustion, it is advisable to:
+
+- limit the set of digest-algorithms to the ones that
+  will be effectively used;
+- establish policies to use only the stronger digest-algorithm
+  provided;
```

Comment:
If I'm trying to avoid resource consumption I might prefer a weaker algorithm...

Reply:
my point is about wasting resources to process unneeded checksums eg.  which are purposedly added to waste receiver's cpu time.